reactor-core/src/test/java/reactor/core/publisher/BlockingTests.java

```diff
@@ -111,7 +111,7 @@ public void blockingLastEarlyComplete() {
 	@Test
 	public void blockingLastTimeout() {
 		assertThatIllegalStateException().isThrownBy(() ->
-				Flux.just(1).delayElements(Duration.ofNanos(100))
+				Flux.just(1).delayElements(Duration.ofMillis(100))
```

Comment:
This one particular change seems weird to me. The rest is ok.

Reply:
real time tests that use nanosecond precision for both scheduling (delays) and timeouts are prone to become flakky at the smallest of host hiccups (OS is busy when the timeout is scheduled, for instance). Having two different orders of magnitude ensures that such a hiccup during block wouldn't result in a test failure.